src/styles/blocks/card.scss

```diff
@@ -0,0 +1,89 @@
+@import "../utils/variables.scss";
+
+//to variables: colors, margins, container-width(calculating for 4 in a row)
```

Comment:
Do not leave comments in the code, if your code needs additional explanation you should think if your code good